openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -417,3 +420,25 @@ export function initBorrowAndReadLinks() {
 
     /* eslint-enable no-unused-vars */
 }
+
+export function initPreviewButton() {
+    /**
+     * Colorbox modal + iframe for Book Preview Button
+     */
+    $(document).ready(function(){
```

Comment:
You no longer need to call `$(document).read(`; just put the stuff in the function here directly.

Reply:
done